modules/vaos/app/services/vaos/systems_service.rb

```diff
@@ -18,10 +18,11 @@ def get_systems(user)
       end
     end
 
-    def get_facilities(user, facility_code)
+    def get_facilities(user, facility_codes)
       with_monitoring do
         url = '/var/VeteranAppointmentRequestService/v4/rest/direct-scheduling/parent-sites'
-        response = perform(:get, url, { 'facility-code' => facility_code }, headers(user))
+        options = { params_encoder: Faraday::FlatParamsEncoder }
```

Comment:
I take it this is needed because of how the array of facility codes deviates from the Rails []= convention? I'm glad no monkey patching was necessary!